### ISSUE TITLE
Entity [...] implements HVACMode(s) [...] without setting the proper ClimateEntityFeature

### ISSUE THREAD

**Issue by tom3q** [scottyphillips/echonetlite_homeassistant#166]
I'm getting the following warnings after updating to HA 2024.2:

```
Entity None (<class 'custom_components.echonetlite.climate.EchonetClimate'>) does not set ClimateEntityFeature.TURN_OFF but implements the turn_off method. Please create a bug report at https://github.com/scottyphillips/echonetlite_homeassistant/issues
Entity None (<class 'custom_components.echonetlite.climate.EchonetClimate'>) does not set ClimateEntityFeature.TURN_ON but implements the turn_on method. Please create a bug report at https://github.com/scottyphillips/echonetlite_homeassistant/issues
Entity None (<class 'custom_components.echonetlite.climate.EchonetClimate'>) implements HVACMode(s): heat, cool, dry, fan_only, heat_cool, off and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/scottyphillips/echonetlite_homeassistant/issues
```

Probably a simple change that I could make myself, but let me file an issue for it first.

**Comment by tom3q** [comment]
Thanks for the quick fix! I can confirm that the warning is no longer present with the latest version.

**Comment by generically-named** [comment]
Hi, unsure if this was resolved in v3.7.9? Tom has stated the warnings are no longer present with the latest version however I am still seeing the warnings after having installed the latest version from HACS. Please see logs below, apologies if this is something on my end.
`Logger: homeassistant.components.climate
Source: components/climate/__init__.py:354
Integration: Climate (documentation, issues)
First occurred: 14:13:31 (3 occurrences)
Last logged: 14:13:31

Entity None (<class 'custom_components.echonetlite.climate.EchonetClimate'>) does not set ClimateEntityFeature.TURN_OFF but implements the turn_off method. Please create a bug report at https://github.com/scottyphillips/echonetlite_homeassistant/issues
Entity None (<class 'custom_components.echonetlite.climate.EchonetClimate'>) does not set ClimateEntityFeature.TURN_ON but implements the turn_on method. Please create a bug report at https://github.com/scottyphillips/echonetlite_homeassistant/issues
Entity None (<class 'custom_components.echonetlite.climate.EchonetClimate'>) implements HVACMode(s): heat, cool, dry, fan_only, heat_cool, off and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/scottyphillips/echonetlite_homeassistant/issues`

**Comment by nao-pon** [comment]
@generically-named This issue will be fixed in the next release version 3.8.0. You can check working with master version before release. master can be installed from the HACS re-download menu.
